cypress/integration/examples/cypress_api.spec.js

```diff
@@ -0,0 +1,219 @@
+/// <reference types="cypress" />
+
+context('Cypress.Commands', () => {
+  beforeEach(() => {
```

Comment:
The base URL in `cy.visit()` should point to the Book Store app, not the Cypress example site.